looprpc/server.proto

```diff
@@ -72,6 +72,9 @@ message ServerLoopOutResponse {
     bytes sender_key = 3;
 
     int32 expiry = 4;
+
+    // A human-readable message for the user of the loop server.
```

Comment:
Looking at this from the client repo, I think "message from the loop server" makes more sense. But really semantics here.

Reply:
fixed